test/Microsoft.Azure.Devices.Edge.Test/Module.cs

```diff
@@ -12,24 +12,25 @@ namespace Microsoft.Azure.Devices.Edge.Test
 
     public class Module : ManualProvisioningFixture
     {
-        private string GetTempSensorImage()
+        private (string Name, string Image) GetTempSensorImage()
```

Comment:
Name and Image should be lowercased (technically camelCase).

Reply:
...but actually, we have a problem here where the name "tempSensor" will be used for three different modules on a single device (when all of the tests are run together), which IoT Hub doesn't like (at least, not within such a short period of time) and which might cause test failures. So this method probably needs to go away, replaced by a class-level const for the tempSensor image, and then unique names for the tempSensor identity in each test.